ftw/simplelayout/js/src/simplelayout/Block.js

```diff
@@ -56,6 +56,20 @@ export default function Block(content, type) {
     this.commit();
   };
 
+  this.updateToolbar = function() {
+    // check if configuration on block indicates that button is active
+    $(".block-server-action", this.toolbar.element).toArray().forEach(function(e) {
+      var buttonParameter = $(e).data();
+      var blockConfig = this.data("config") || {};
+
+      var active = Object.keys(buttonParameter).every(function(propertyName) {
+        return (propertyName in blockConfig &&
+                buttonParameter[propertyName] == blockConfig[propertyName]);
```

Comment:
Always use `===` for comparison in javascript. This prevents the engine from type converting.